hook.php

```diff
@@ -357,7 +357,12 @@ function plugin_fields_addWhere($link, $nott, $itemtype, $ID, $val, $searchtype)
             ],
         )
     ) {
-        return $link . $DB->quoteName("$table" . '_' . "$field") . '.' . $DB->quoteName($field) . 'LIKE ' . $DB->quoteValue("%\"$val\"%") ;
+        switch ($searchtype) {
+            case 'equals':
+                return $link . $DB->quoteName("$table" . '_' . "$field") . '.' . $DB->quoteName($field) . 'LIKE ' . $DB->quoteValue("%\"$val\"%") ;
+            case 'notequals':
+                return $link . $DB->quoteName("$table" . '_' . "$field") . '.' . $DB->quoteName($field) . 'LIKE ' . $DB->quoteValue("%\"$val\"%") . ' OR ' . $link . $DB->quoteName("$table" . '_' . "$field") . '.' . $DB->quoteName($field) . 'IS NULL ';
```

Comment:
```suggestion
                return $link . $DB->quoteName("$table" . '_' . "$field") . '.' . $DB->quoteName($field) . ' NOT LIKE ' . $DB->quoteValue("%\"$val\"%") . ' OR ' . $link . $DB->quoteName("$table" . '_' . "$field") . '.' . $DB->quoteName($field) . 'IS NULL ';
```
Should be NOT LIKE no ?